src/Core/Domain/Product/Combination/CommandHandler/UpdateCombinationFromListingHandlerInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\Combination\CommandHandler;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\Combination\Command\UpdateCombinationFromListingCommand;
+
+/**
+ * Defines contract to handle @see UpdateCombinationFromListingCommand
```

Comment:
```suggestion
 * Defines contract to handle @see UpdateCombinationFromListingCommand
 *
 * To update combination from listing is the action triggered by a Back Office user from the combinations listing interface.
```
I'l wondering if we should start to be more descriptive? 🤔 

Some Queries/Commands have very explanatory names, they dont need descriptions. But others are more ambiguous.

Reply:
In this case I agree a little bit more description is useful

Reply:
up until now I thought we describe what command does in the command (not in handler).
UpdateCombinationFromListingCommand has this comment `Updates combination in list of combinations`. Isn't that enough? Should we start describing usecases inside Handlers from now on?